swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
```

Comment:
Nice job setting the default value of inventory to None since default parameters should not be initialized to empty mutable datatypes!